UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helpers/NMHelperTest.cs

```diff
@@ -0,0 +1,425 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UKHO.FileShareClient.Models;
+using UKHO.MaritimeSafetyInformation.Common.Helpers;
+using UKHO.MaritimeSafetyInformation.Common.Models.NoticesToMariners;
+
+namespace UKHO.MaritimeSafetyInformation.Common.UnitTests.Helpers
+{
+    public class NMHelperTest
+    {
+        [Test]
+        public void WhenNMHelperCallsListFilesResponse_ThenConversionIsCorrect()
+        {
+            BatchSearchResponse SearchResult = SetSearchResultForWeekly();
```

Comment:
Camel case variable names please:
```suggestion
            BatchSearchResponse searchResult = SetSearchResultForWeekly();
```